Source/JavaScriptCore/jsc.cpp

```diff
@@ -808,6 +809,10 @@ class GlobalObject final : public JSGlobalObject {
         }
 
         if (Options::useDollarVM()) {
+#if PLATFORM(COCOA)
+            enableAllSDKAlignedBehaviors();
+#endif
```

Comment:
Why is `enableAllSDKAlignedBehaviors()` only needed for the jsc shell binary in testing mode?  Why not all the time?

Reply:
Fixed!